packages/salesforcedx-vscode-apex/resources/dark/testSkip.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 114 114"><defs><style>.cls-1{fill:#ff8b00;}</style></defs><title>Artboard 1 copy</title><path class="cls-1" d="M57,7.5A49.5,49.5,0,1,0,106.5,57,49.5,49.5,0,0,0,57,7.5Zm0,87.42A37.93,37.93,0,1,1,94.93,57,37.93,37.93,0,0,1,57,94.93Z"/><circle class="cls-1" cx="57" cy="57" r="25.88"/></svg>
```

Comment:
Are these testSkip.svg files just copies of similar svg's in the project and you just manually updated their style ?

Reply:
Yeah, they're copies of the other svg's we already had in the project and I manually updated the colors. Is there a different way I should do it?

Reply:
I thank that's ok